content/en/docs/distributions/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -130,9 +133,116 @@ kfctl apply -V -f ${CONFIG_FILE}
+- ../distributions/stacks/ibm/application/cert-manager-kube-system-resources
+- ../distributions/stacks/ibm/application/cert-manager-crds
+- ../distributions/stacks/ibm/application/cert-manager
+# Istio (requires)
+- ../distributions/stacks/ibm/application/istio-1-9-0
+# OIDC Authservice (required)
+- ../distributions/stacks/ibm/application/oidc-authservice
+- ../distributions/stacks/ibm/application/dex-auth
+- ../distributions/stacks/ibm/application/user-namespace
+# KNative
+- ../distributions/stacks/ibm/application/knative
+# Kubeflow namespace (required)
+- ../distributions/stacks/ibm/application/kubeflow-namespace
+# Kubeflow Roles (required)
+- ../distributions/stacks/ibm/application/kubeflow-roles
+# Kubeflow istio resource (required)
+- ../distributions/stacks/ibm/application/kubeflow-istio-resources
+# Kubeflow Pipelines Argo
+# - ../distributions/stacks/ibm/application/kfp-argo
+# Kubeflow Pipelines Tekton
+- ../distributions/stacks/ibm/application/kfp-tekton
+# KFServing
+- ../distributions/stacks/ibm/application/kfserving
+# Katib
+- ../distributions/stacks/ibm/application/katib
+# Central Dashboard (required)
+- ../distributions/stacks/ibm/application/centraldashboard/single-user
+# Admission Webhook (required)
+- ../distributions/stacks/ibm/application/admission-webhook
+# Notebook Controller
+- ../distributions/stacks/ibm/application/notebook-controller
+# Jupyter Web App
+- ../distributions/stacks/ibm/application/jupyter-web-app/insecure
+# Profiles + KFAM (required)
+- ../distributions/stacks/ibm/application/profiles
+# Volumes Web App
+- ../distributions/stacks/ibm/application/volumes-web-app/insecure
+# Tensorboards Web App
+-  ../distributions/stacks/ibm/application/tensorboard-controller
+# Tensorboard Controller
+-  ../distributions/stacks/ibm/application/tensorboard-web-app/insecure
+# TFJob Operator
+- ../distributions/stacks/ibm/application/tf-job
+# Pytorch Operator
+- ../distributions/stacks/ibm/application/pytorch-job
+# MPI Operator
+- ../distributions/stacks/ibm/application/mpi-job
+# MXNet Operator
+- ../distributions/stacks/ibm/application/mxnet-job
+# XGBoost Operator
+- ../distributions/stacks/ibm/application/xgboost-job
+```
+
+> Not all the component are required for running kubeflow. 
+
+5. You can apply the kustomize file from the root of the repositor.
+
+```
+kustomize build -f ibm/kustomization.yaml | kubectl apply -f -
+```
+
+There might be errors the first time we run the command. We can also run this in a while loop until success.
+
+```
+while ! kustomize build ibm | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+```
+
+### Access your cluster
+
+The Kubeflow endpoint is exposed with [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) 30380. If you don't have any experience on Kubernetes, you can [expose the Kubeflow endpoint as a LoadBalancer](#expose-the-kubeflow-endpoint-as-loadbalancer) and access the **EXTERNAL_IP**.
```

Comment:
Suggestion:

- `30380` (with back ticks)

What does "If you don't have any experience on Kubernetes" mean in this context?

Reply:
That sentence was here from before. I will update it with something different.